api/v2/models/bouquets/bouquets_model.py

```diff
@@ -1,29 +1,31 @@
-from sqlalchemy import (Column, String, Integer, Boolean, Enum)
+from sqlalchemy import *
```

Comment:
Why do you introduce wildcard an import here? Please read import best practice here: https://www.python.org/dev/peps/pep-0008/#imports